packages/agents/router/src/config.ts

```diff
@@ -84,6 +84,7 @@ export const NxtpRouterConfigSchema = Type.Object({
   polling: TPollingConfig,
   subgraphPrefix: Type.Optional(Type.String()),
   environment: Type.Union([Type.Literal("staging"), Type.Literal("production")]),
+  nomad_environment: Type.Union([Type.Literal("staging"), Type.Literal("production")]),
```

Comment:
Can we make this camel case?

Reply:
Also we need to add this config item to the below section where it parses it from env. And we need to add the correct mocks for it. Also need to add the config item to the terraform files inside `ops/` to make the live agents use the proper vars.